landing-page/content/common/table.md

```diff
@@ -0,0 +1,72 @@
+
+## Schema
+
+The schema of a table defines the elements contained in a single row. Each field in a schema can be optional or required and has an associated ID, name, and type. Schemas can also include aliases for certain fields. As the schema of an Iceberg table is changed, the new schema is given a unique ID. Past schemas are preserved in the table metadata and allow schemas to be restored when tables are rolled back.
+
+## Sort Order
+
+A table’s sort order is a special property that defines which fields the table’s records should be ordered by when writing data files.  
+
+## Snapshots
+
+Changes in table state are captured as snapshots. Snapshots allow you to query historical versions of a table and fully roll back using metadata-only operations. In essence, a snapshot is a reference to a specific metadata file, and changing the table state is controlled by changing this reference. This reference is maintained by the Iceberg catalog and ties a table reference to its current snapshot.
+
+## Anatomy of a Table
+
+{{< figure src="/img/iceberg-table-structure.png" alt="button" >}}
+Iceberg tables are composed of five types of files: metadata, manifest lists, manifests, data and delete files. All files in an Iceberg table are immutable; making a change to an iceberg table involves adding to or removing from the set of files which make up the table and reflecting those changes in a new metadata JSON file for the table. When reading and writing Iceberg tables, you don’t need to be aware of these internal details, but they’re used the compute engines to expose all of Iceberg’s powerful features.
+
+## Metadata JSON ([spec](../../spec/#table-metadata))
+
+The top level object, simply referred to as the table metadata file, is a single json file that includes high level information on various aspects of a table such as current and past schemas, partitioning, and configured sort-orders. Table properties are also included in the table metadata file. When changes are made to a table, a new metadata file is created and the table’s entry in the Iceberg catalog is updated to point to this new file.
+
+The metadata file also contains a reference to all of the table’s snapshots which provides the location of each snapshots manifest list.
+
+## Manifest Lists ([spec](../../spec/#manifest-lists))
+
+Each table snapshot has a corresponding manifest list. The manifest list serves as a high-level index on the data for a particular snapshot. This file contains references to all of the manifest files that are part of the snapshot along with relevant summary metrics.
```

Comment:
I may not say "high-level index on the data". I'd probably be more specific and say that it tracks the partition ranges that are in each manifest so that it can skip entire manifest files when the query doesn't use those partitions.